src/components/ConcentratedLiquidity/PositionCard/index.tsx

```diff
@@ -12,13 +16,90 @@ import {
   OptionButton,
   OptionsWrapper,
   Panel,
-  Price,
+  PriceWrapper,
   Row,
 } from './styles';
 import { PositionCardProps } from './types';
 
+export function getPriceOrderingFromPositionForUI(position?: Position): {
```

Comment:
lets move this function to `src/components/ConcentratedLiquidity/PositionCard/helpers.tsx`